tests/cases/conformance/salsa/typeFromPropertyAssignment20.ts

```diff
@@ -0,0 +1,20 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @Filename: bluebird.js
+
+!function outer (f) { return f }(
+    function inner () {
+        function Async() {
+            this._trampolineEnabled = true;
+        }
+
+        Async.prototype.disableTrampolineIfNecessary = function dtin(b) {
+            if (b) {
+                this._trampolineEnabled = false;
+            }
+        };
+        var a = new Async()
```

Comment:
Are these 3 lines necessary to the test?

Reply:
No, this really only checks (1) lack of crashes and (2) symbol correctness. Removed.